### ISSUE TITLE
crash with errors

### ISSUE THREAD

**Issue by rodgomesc** [spyoungtech/mirror-action#5]
![image](https://user-images.githubusercontent.com/4893591/80996534-541f8d00-8e16-11ea-9235-f3cfc41b13cd.png)

my yml file:

```yml
name: Release App Version

on:
  push:
    branches:
      - master

jobs:
  release:
    name: Release
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@master
      - uses: actions/setup-node@v1
        with:
          node-version: '12.x'
      - run: npx semantic-release
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

  mirrorToGitLab:
    name: Mirror to GitLab
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@master
      - name: Push to GitLab
        uses: spyoungtech/mirror-action@master
        env:
          GIT_USERNAME: rodgomesc
          GIT_PASSWORD: ${{ secrets.GITLAB_PASSWORD }}
        with:
          args: https://gitlab.com/Siedos/siedos.git


```

**Comment by spyoungtech** [comment]
Sorry, the existing examples haven't been updated for GitHub Actions 2.0

**Comment by spyoungtech** [comment]
This has been fixed now @rodgomesc -- There was an issue when using `@master` -- `@v0.2.2` would also work for you if you specified source repo like in the docs.

Though, the preferred method of configuring this project has changed. See the readme for more details.

I've also tried to make things as backwards-compatible as possible. I recommend using the new latest version on the marketplace: `@0.3.0`
